extensions/amp-iframe/amp-iframe.md

```diff
@@ -250,6 +250,49 @@ The intersection message would be sent by the parent to the iframe when the
 iframe moves in or out of the viewport (or is partially visible), when the
 iframe is scrolled or resized.
 
+## Iframe & Consent Data
+
+Iframes can send a `send-consent-data` message to receive consent data if a CMP is present on their parents page.
+
+_Note: In the following examples, we assume the script is in the created iframe, where `window.parent` is the top window. If the script lives in a nested iframe, change `window.parent` to the top AMP window._
+
+_Example: iframe `send-consent-data` request_
+
+```javascript
+window.parent.postMessage(
+  {
+    sentinel: 'amp',
+    type: 'send-consent-data',
+  },
+  '*'
+);
+```
+
+The iframe can receive the consent data response by listening to the `consent-data` message.
+
+**Please note:**
+
+- The `consent-data` response will only be sent once and won't update the iframe when the consent state changes (for example when the user decides to reject consent by using the post prompt ui)
+- The response is delayed until the user interacts with the amp-consent ui in case iframe loading is not blocked by using `data-block-on-consent`
```

Comment:
The response is not delayed until the user interact. It is delayed based on the `data-block-on-consent` value, and it's corresponding setting. 
For example one can customize the consent service to resolve after a 2 seconds timeout. In this case, the response will resolve after 2 seconds, not wait for the user interaction.

Reply:
Thanks for clarifying!